README.md

```diff
@@ -798,6 +801,17 @@ const view = instance.views.getView('zoomview');
 view.showPlayeadTime(false); // Remove the time from the playhead marker.
 ```
 
+## Cue events
+
+Emit events when the playhead reaches a point or segment boundary:
+
+```js
+const peaks = Peaks.init({ ..., emitCueEvents: true });
+peaks.on('points.enter', function(point, time, id) { /* ... */ });
```

Comment:
The `time` and `id` parameters aren't needed here, as you can get at them via the `point` or `segment` object.